internal/pkg/warewulfd/provision.go

```diff
@@ -103,7 +107,8 @@ func ProvisionSend(w http.ResponseWriter, req *http.Request) {
 			Hwaddr:         rinfo.hwaddr,
 			ContainerName:  node.ContainerName.Get(),
 			KernelArgs:     node.Kernel.Args.Get(),
-			KernelOverride: node.Kernel.Override.Get()}
+			KernelOverride: node.Kernel.Override.Get(),
+			NetDevs:        tstruct.NetDevs}
```

Comment:
@JasonYangShadow can I get you to add a test for my changes here?

Reply:
@anderbubble sure will do that

Reply:
I've pushed test cases in this commit
https://github.com/warewulf/warewulf/pull/1191/commits/75c70fe3e2893808f9bff97049a1cb1b434c55b2

Reply:
Thanks!